package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.10.1-beta.0",
+  "version": "0.10.0-beta.0",
```

Comment:
Why the version revert here? Isn't `0.10.1-beta.0` published? Complete newbie here 😅

Reply:
I was looking into the same, this is a bug (!) with the script to revert a dev version, and with the lint to check if the current version is okay (if you notice the pipeline passes when it should fail https://github.com/remoteoss/json-schema-form/actions/runs/10096419678/job/27919143600?pr=79)

The ideal flow:
1. The PR is updated with the main (have the latest version).
2. You release a dev version
3. The CI lint fails because the version is not a dev one.
4. You revert the dev version (it's a script that reverts the file change)
5. The CI lint passes because the version is restored.
6. You merge the PR and release a new version (beta) in main.


This flow:
1. The PR is outdated with main (it is still 10.0 instead of 10.1)
2. You release a dev version 
3. The CI lint fails because the version is not a dev one.
4. You revert the dev version (it's a script that reverts the file change)
5. The CI lint passes because the version is restored. (bug! 🐛 it should fail instead because the version was changed. Or at least fail because the MR is not updated with main) 
6. You merge the PR and release a new version (beta) in main.

Is this clear? 😅 Anyway, I manually fixed the version (rebased with main) and will open an issue to fix the CI lint scripts.

Reply:
Actually... this branch was already updated with main (10.1.0), so it's a bug with the step 4 (revert the dev version!) (commit https://github.com/remoteoss/json-schema-form/pull/79/commits/62b6b27982fb2b34371642d65c642aba875acfcd). Anyway, a problem for another day.

Reply:
In the meantime, https://github.com/remoteoss/json-schema-form/pull/81 just got merged, it has conflicts, let me fix them in a minute.

Reply:
Yes, probably. I added this thread as reference for when someone picks it up :)